server/app/graphql/types/query_type.rb

```diff
@@ -13,6 +13,98 @@ class QueryType < Types::BaseObject
     field :categories, resolver: Resolvers::Categories
     field :interaction_claim_types, resolver: Resolvers::InteractionClaimTypes
 
+    field :drug_suggestions, [Types::DrugSuggestionType], null: true do
+      description "A searchable drug name or alias that can be completed from the supplied term"
+      argument :term, String, required: true
+      argument :n, Int, required: false
+    end
+
+
+    def drug_suggestions(term:, n: 10)
+      matches = Drug.where("drugs.name ILIKE ?", "#{term}%").limit(n).map do |drug|
+        {
+          "drug_name": drug.name,
```

Comment:
Really small nitpick, the performance difference will be negligible in real life, but idiomatically you'd use symbols here for the dictionary keys, not strings, as each iteration will `new` up the strings again.

```ruby
{
   drug_name: drug.name,
   concept_id: drug.concept_id,
   suggestion: drug.name,
   suggestion_type: 'NAME'
}
```
and so on

Reply:
Python brain at work, thanks